apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -47,6 +47,15 @@ function showHomepage() {
         />
       )}
 
+      {(!isTeacher && showUiTips) && (
+        <UiTips
+          tips={
+            [
+              {type: "triggered", position: {top: 80, right: 15}, text: i18n.homepageUiTipAlreadyHome(), triggerId: "logo_home_link", arrowDirection: "up_corner"}
+            ]}
+        />
+      )}
```

Comment:
Yeah I just started putting those parens in as a habit because I was bitten by needing them when there was an `II` in the expression.